build/ublue-os-just/ublue-os-just.sh

```diff
@@ -1,10 +1,11 @@
-# Add uBlue's justfiles to user HOME if they don't already have a justfile.
+# Add uBlue's justfiles to users with home directories which lack a justfile.
 
-UBLUE_JUST=/usr/share/ublue-os/just
-USER_JUSTFILE="${HOME}/.justfile"
-if [ ! -f "${USER_JUSTFILE}" ]; then
+if [ ! -z "$HOME" ] && [ -d "$HOME" ] && [ ! -f "${HOME}/.justfile" ]; then
+  UBLUE_JUST=/usr/share/ublue-os/just
+  USER_JUSTFILE="${HOME}/.justfile"
+  touch "${USER_JUSTFILE}"
   if [ -f ${UBLUE_JUST}/main.just ]; then
-    cp ${UBLUE_JUST}/main.justfile "${USER_JUSTFILE}"
```

Comment:
Nice. It's easy to get blind from these filenames since we're writing `.just` files to `.justfile`, so I didn't notice that either!